src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -11,11 +11,13 @@
 using NuGet.Packaging.Core;
 using NuGet.ProjectModel;
 using NuGet.Protocol;
+using NuGet.Versioning;
 
 namespace NuGet.Commands
 {
     public class NoOpRestoreUtilities
     {
+        public static readonly string NoOpCacheFileName = "project.assets.nuget.cache";
```

Comment:
Are there any tools depending on the previous file name? 
How the new name helps to improve the no-op performance? (Maybe add few comments ? )
If the cache file name and the project.assets .json needs to be in sync maybe the string prefix "project.assets" can be shared from a common location?

Reply:
> Are there any tools depending on the previous file name?

I was told that the file and its layout is proprietary and no one should have taken a dependency on it.  As long as they are using our API to read it, they'll be insulated.

> How the new name helps to improve the no-op performance?

We are currently reading the assets file to determine if two projects are sharing the same output.  However, I'm now storing the full path in the cache file as well but since the cache file is unique per project name, no-op would work differently in the case when you have two projects with different names sharing the same output folder.  Renaming this file keeps behavior.  I did type that up the main comment but maybe its not clear.

> If the cache file name and the project.assets .json needs to be in sync maybe the string prefix "project.assets" can be shared from a common location?

Possibly, I just picked the name out of a hat.  The NuGet team will need to tell me if A) the name is good and B) should I add a common prefix const somewhere

Reply:
The name looks fine to me. 
If the name it needs to be in sync with project.assets.json
1. the common prefix could be added somewhere at the NuGet.ProjectModel level but I could not find an existent good place to put it. 
2. another option would be to add an unit test.